tools/autograd/templates/python_variable_methods.cpp

```diff
@@ -520,26 +520,31 @@ static PyObject * THPVariable_type(PyObject* self, PyObject* args, PyObject* kwa
 {
   HANDLE_TH_ERRORS
   static PythonArgParser parser({
+    "type(Type dtype, bool non_blocking=False)",
```

Comment:
Can you merge this with the overload below? Having only one (non-deprecated) overload gives better error messages